source/exe/BUILD

```diff
@@ -93,6 +118,14 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
+    name = "envoy_stripped_main_base_lib",
+    deps = [
+        ":stripped_main_base_lib",
+        "//source/common/version:version_linkstamp",
```

Comment:
nit: from a quick glance looking at the `version_linkstamp` build target, it's not clear to me what it does, and so it's not clear to me the difference between `stripped_main_base_lib` and `envoy_stripped_main_base_lib`.. perhaps a comment would help?